### PR TITLE
[Communication] Sms re-design

### PR DESCRIPTION
**Pull request by paolamvhz**
We are implementing the broadcast sms.

Scenarios 1:1
1:N

Adding a new Tag in the SmsOptions

### CODE REVIEW

**Review by beltr0n — COMMENTED** [review]
Left some comments, I also noticed there are some formatting differences scattered across the PR. Is there a formatting standard?

File: sdk/communication/azure-communication-sms/src/main/java/com/azure/communication/sms/models/SmsSendResult.java [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        return this;
+    }
+
+    /**
+     * Get the repeatabilityResult property: The result of a repeatable request with one of the case-insensitive values
+     * accepted or rejected.
+     *
+     * @return the repeatabilityResult value.
+     */
+    public SmsSendResultRepeatabilityResult getRepeatabilityResult() {
```
[the remark this answers is not in the document]
Comment (paolamvhz) [reply sdk/communication/azure-communication-sms/src/main/java/com/azure/communication/sms/models/SmsSendResult.java]: commented at this momment

File: sdk/communication/azure-communication-sms/src/main/java/com/azure/communication/sms/SmsAsyncClient.java [hunk 2]
```diff
@@ -38,117 +38,121 @@
...
-
-        return sendMessageRequest;
+
+    private SendMessageRequest createSendMessageRequest(String from, Iterable<String> smsRecipient, String message,
+                                                        SmsSendOptions smsOptions) {
+        SendMessageRequest request = new SendMessageRequest();
+        List<SmsRecipient> recipients = new ArrayList<SmsRecipient>();
+        for (String s : smsRecipient) {
+            recipients.add(new SmsRecipient().setTo(s));
+
```
[the remark this answers is not in the document]
Comment (paolamvhz) [reply sdk/communication/azure-communication-sms/src/main/java/com/azure/communication/sms/SmsAsyncClient.java]: fixed

File: sdk/communication/azure-communication-sms/README.md [hunk 3]
```diff
@@ -96,32 +74,60 @@ SmsClient smsClient = new SmsClientBuilder()
...
+// Send the message to a list of  phone Numbers and check the response for a messages ids
+SmsSendResult response = smsClient.send(
+    "<leased-phone-number>",
+    to,
     "your message",
     options /* Optional */);
 
 System.out.println("MessageId: " + response.getMessageId());
 ```
+### Sending a message to multiples recipients
```
[the remark this answers is not in the document]
Comment (paolamvhz) [reply sdk/communication/azure-communication-sms/README.md]: fixed

**Comment by paolamvhz** [comment]
erro branch

**Pull request closed without merging**